src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Unreleased
 
+* `ActivityProcessor` implements IDisposable
```

Comment:
We need more detailed version of whats changing here.
1. This is not a breaking change as such - because the ActivityProcessor provides a default IDisposable implementation which calls ShutDown.
2. User's writing custom ActivityProcessor may override the Dispose behavior to do more.
3. BatchingProcessor/SimpleProcessor is disposable and it disposes the containing exporter

Reply:
just updated. let me know what do u think.